.coveragerc

```diff
@@ -1,2 +1,2 @@
-[run]
-omit = */tests/*, */migrations/*, */urls.py, */settings/*, */wsgi.py, department/manage.py, department/setup.py, */asgi.py
\ No newline at end of file
+[run]
+omit = */migrations/*, */urls.py, */settings.py, */wsgi.py, department/manage.py, department/setup.py, */asgi.py

```

Comment:
Please include the tests folder as well.

Reply:
Added